app/models/ipeds_ic_ay.rb

```diff
@@ -2,6 +2,8 @@
 class IpedsIcAy < ActiveRecord::Base
   include Loadable, Exportable
 
+  USE_COLUMNS = [:tuition_in_state, :tuition_out_of_state, :books].freeze
```

Comment:
E.g. maybe a mixin that generates `columns`, assuming the table naming is consistently discoverable from the ActiveModel class, which I think it should be.